recipes/libavif/build.sh

```diff
@@ -0,0 +1,16 @@
+mkdir build
+cd build
+
+# Other codecs cannot be enabled because they are not on conda-forge
+cmake .. -GNinja \
+-DCMAKE_INSTALL_PREFIX="$PREFIX" \
```

Comment:
It could be useful to add `${CMAKE_ARGS}` here to enable cross-compiling easily in the future.

Reply:
Thanks for the reminder! If this was in the CMake section of the conda-forge docs in addition to the CUDA section, I would have remembered!